.github/actions/simple-md-toc/action.yml

```diff
@@ -0,0 +1,57 @@
+#Github Composite actions to generate Markdown Table of Contents
```

Comment:
```suggestion
```

whatever needed should be in the description

Reply:
Oops sorry that was my prompt to get the skeleton :p